micro-demos/services-yaml/demo.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
```

Comment:
I am not sure I see the value of this one.  I don't really want to keep them in sync.  Do we really need it?